### PR TITLE
docs: use enum type instead of string

### PR DESCRIPTION
**Pull request by MarcusNotheis**
In several places, `string` was used as `@type` annotation for the documentation. This PR is replacing those string annotations with the corresponding enums.

### CODE REVIEW

**Comment by MarcusNotheis** [comment]
please do not merge yet, just found another one

File: packages/main/src/StandardListItem.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ const metadata = {
 		 * Defines the state of the <code>info</code>.
 		 * <br>
 		 * Available options are: <code>"None"</code> (by default), <code>"Success"</code>, <code>"Warning"</code> and <code>"Erorr"</code>.
-		 * @type {string}
+		 * @type {ValueState}
```
Comment (ilhan007) [inline packages/main/src/StandardListItem.js]: can you also add the @ defaultvalue None

File: packages/main/src/SuggestionItem.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ const metadata = {
 		 * Defines the state of the <code>info</code>.
 		 * <br><br>
 		 * Available options are: <code>"None"</code> (by default), <code>"Success"</code>, <code>"Warning"</code> and <code>"Erorr"</code>.
-		 * @type {string}
+		 * @type {ValueState}
```
Comment (ilhan007) [inline packages/main/src/SuggestionItem.js]: can you also add the @ defaultvalue None

and below for the "group" @ defaultvalue dalse

File: packages/main/src/MessageStrip.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ const metadata = {
 		 *
 		 * See all the available icons in the <ui5-link target="_blank" href="https://openui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html" class="api-table-content-cell-link">Icon Explorer</ui5-link>.
 		 *
-		 * @type {string}
+		 * @type {HTMLElement}
```
Comment (ilhan007) [inline packages/main/src/MessageStrip.js]: can you please add @ slot and remove the @ defaultvalue

File: packages/main/src/StandardListItem.js (hunk 1 above)
Comment (MarcusNotheis) [reply packages/main/src/StandardListItem.js]: done

File: packages/main/src/SuggestionItem.js (hunk 2 above)
Comment (MarcusNotheis) [reply packages/main/src/SuggestionItem.js]: done

File: packages/main/src/MessageStrip.js (hunk 3 above)
Comment (MarcusNotheis) [reply packages/main/src/MessageStrip.js]: done. is `@type HTMLElement` fine or should I use `Node` here as for the default slot?

Comment (ilhan007) [reply packages/main/src/MessageStrip.js]: HTMLElement is fine

**Pull request merged**